### DISCUSSION TITLE
Ability to sort item by dragging with mouse

### DISCUSSION THREAD

**Discussion by wssrandd** [Hofknecht/SystemTrayMenu#404] [Q&A] [answered]
This is a slick tool that helps restore functionality lost in Windows 11. In previous Windows versions, you could drag items in a toolbar to have a custom sort order, versus alphabetical.

Is it possible to implement this in a future release?

Thanks.

**Comment by Hofknecht** [comment]
@wssrandd 
Thank you for the feedback!
We have a issue therefore, see [Feature] make results sortable manually #306
i added a comment there that you voted for it. 
We're definitely still implementing that, but I want to do a few other things before that

**Comment by jeremymbwilson** [comment]
Firstly - great tool !  Gives me (almost) all the functionality that I need.  Really great job !

I would love to be able to sort manually by dragging and dropping.  This was suggested around 2 years ago, now, and I was wondering if there is any progress with this.  I understand that you have higher priorities to deal with, but would love to see this implemented, if possible.

Keep up the great work !

**Comment by explor202** [comment, marked as answer]
I agree, manual sorting is ideal for me too. However, a quick workaround is to use a utility that can modify the date of each file. I just used [Attribute Changer](https://www.petges.lu/download/) to edit the "Date Modified" data and now everything is sorted in the order I prefer. I changed the date to 11/15/2024, then changed the time in five minute increments, starting at 07:00:00. I prefer bottom up selecting of my files, so the first file I need to select has the oldest date and appears at the bottom of the list. 

NOTE: Depending on the age of other files that are in the same folder, you may end up having to modify the date on all of them in order to get the correct ordering you want.

**Reply by sweetman-zz** [reply]
I add my 2c:
why not enable a sort based on most used items? so, the tools/links I use the most are always at hand.
tony
